ykman/scancodes/__init__.py

```diff
@@ -27,16 +27,17 @@
 
 from __future__ import absolute_import
 from enum import Enum
-from . import us, de, fr, it, modhex, norman
+from . import us, de, fr, it, modhex, norman, bepo
```

Comment:
Is it time to sort these? :smile:

Reply:
We don't currently sort imports in anyway right? If we decide to I think we should do it across the project. I looked quickly if Black supports this, but doesn't look like it https://github.com/psf/black/issues/333

Reply:
Not really, no. I've been doing it for new imports but it's indeed not uniform across the project. There are tools like `isort` for doing it automatically if we want to, but never mind for now.